.changeset/grumpy-ads-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/select": minor
```

Comment:
I think this should be a `patch` since a `portal` isn't a feature anyone can use, rather it solves an implementation issue.